web/src/click_handlers.ts

```diff
@@ -983,6 +983,33 @@ export function initialize(): void {
         }
     });
 
+    const compose_buttons = tippy.default(".compose_button_tooltip", {
+        trigger: "mouseenter",
+        placement: "top",
+    });
+
+    tippy.createSingleton(compose_buttons, {
+        onTrigger(instance, event) {
+            const currentTarget = event.currentTarget;
+            if (currentTarget instanceof HTMLElement) {
+                const templateId = currentTarget.dataset.tooltipTemplateId;
+                if (templateId) {
+                    const templateElement = document.querySelector(`#${templateId}`);
+                    if (templateElement) {
+                        const parsedHTML = ui_util.parse_html(templateElement.innerHTML);
+                        instance.setContent(parsedHTML);
+                    }
+                }
```

Comment:
Lets use `get_tooltip_content` from `web/src/tippyjs.ts` instead of redefining an almost identical logic here again.